boat-maven-plugin/src/main/java/com/backbase/oss/boat/GenerateFromDirectoryDocMojo.java

```diff
@@ -29,19 +39,19 @@ public void execute() throws MojoExecutionException {
         throw new MojoExecutionException("No OpenAPI specs found in: " + inputSpec);
       }
 
-      for(File f : inputSpecs){
+      for (File f : inputSpecs) {
         inputSpec = f.getPath();
-        output = new File(outPutDirectory.getPath(),f.getName().substring(0,f.getName().lastIndexOf(".")).concat("-docs"));
+        output = new File(outPutDirectory.getPath(), f.getName().substring(0, f.getName().lastIndexOf(".")).concat("-docs"));
```

Comment:
Output are always docs? Also for client generation?

Reply:
The directory input was only wanted for doc goal

Reply:
I can change it for others if wanted

Reply:
Ah! I see that now! Sorry. Disregard the previous comment :)